apps/varnish.go

```diff
@@ -52,3 +53,35 @@ func (r MetricsReceiverVarnish) Pipelines() []otel.Pipeline {
 func init() {
 	confgenerator.MetricsReceiverTypes.RegisterType(func() confgenerator.Component { return &MetricsReceiverVarnish{} })
 }
+
+type LoggingProcessorVarnishAccess struct {
+	confgenerator.ConfigComponent `yaml:",inline"`
+}
+
+func (LoggingProcessorVarnishAccess) Type() string {
+	return "varnish_access"
```

Comment:
I don't see any other varnish logs, is there a reason this isn't just `varnish`? Are there other types of logs that we aren't reporting/collecting yet?

Reply:
updated to `varnish` since it is the only log file